package.json

```diff
@@ -126,7 +126,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "web-scripts precommit"
+      "pre-commit": "web-scripts precommit --no-typecheck"
```

Comment:
This flag is needed as of [`@spotify/web-scripts v7.0.0`](https://github.com/spotify/web-scripts/releases/tag/v7.0.0)